packages/store-ui/src/organisms/SearchBar/SearchBar.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+
+import SearchBar from './SearchBar'
+
+describe('SearchBar', () => {
```

Comment:
Can add test for more cases, like search bar with input and see the popover; see if the children you pass is being rendered; test if the onSubmit is being called.